.github/workflows/monitor.image+upload.yaml

```diff
@@ -9,6 +9,12 @@ on:
     types:
       - closed
 
+  push:
+    branches:
+      - i-720/api-for-orak-network-front 
```

Comment:
@joseph-bisonai I think this specific branch name was only for testing. Should we remove it now?

Reply:
sure this is for my branch to test it. need to remove it